dotenv/lupus-ci.env

```diff
@@ -0,0 +1,15 @@
+PHAPP_ENV_TYPE=lagoon
+PHAPP_ENV_MODE=development
+PHAPP_ENV_COLOR=302f2f
+
+# Adjust available URLs while testing.
+# @see web/sites/sites.php
+
+# Make use of docker compose environment variables defined in .env.
+APP_MULTISITE_DOMAIN=$COMPOSE_HOSTNAME
+# Set cookie_domain service parameter via services_env_parameter module.
+DRUPAL_SERVICE_session__storage__options___cookie_domain=".ci2.drunomics.com"
+
+# ENV_UNIX_GROUP_WEBSERVER="www-data"
+CHROME_HOST=chrome
+BEHAT_EXTRA_ARGUMENTS="--format=pretty --out=std --format=junit --out=behat_results"
```

Comment:
hm, no that file is for internal setup and thus should be added by project-devsetup

Reply:
Good point, already over there.